src/esp/metadata/attributes/AttributesEnumMaps.h

```diff
@@ -8,15 +8,24 @@
 #include "esp/core/Esp.h"
 #include "esp/gfx/LightSetup.h"
 namespace esp {
-namespace asset {
-enum class AssetType;
-}
 namespace physics {
 enum class MotionType;
 }
 namespace metadata {
 namespace attributes {
 
+/**
+ * @brief Supported Asset types
+ */
+enum class AssetType {
+  UNKNOWN,
+  MP3D_MESH,
+  INSTANCE_MESH,
+  UNKNOWN2,
```

Comment:
unknown vs unknown2?

Reply:
That's very old, we don't use it anywhere - i'll get rid of it.